jest.config.legacy.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
         "\\.(gql|graphql)$": "jest-transform-graphql",
         "^.+\\.coffee$":
           "<rootDir>/node_modules/jest-coffee-preprocessor/index.js",
-        ".(ts|tsx|js|jsx)": "babel-jest",
+        "^.+\\.(ts|tsx|js|jsx)$": "babel-jest",
```

Comment:
was this causing issues before?

Reply:
Yes. This setting was leading our json files to be misinterpreted by the transpiler (build would fail trying to read json files as js files).

Example of failure when running `yarn test:ci:jest:v2`:
```
FAIL  src/v2/Apps/BuyerGuarantee/Routes/__tests__/BuyerGuaranteeIndex.jest.tsx
  ● Test suite failed to run

    SyntaxError: /app/src/v2/compiled-lang/en-us.json: Missing semicolon. (2:18)

      1 | {
    > 2 |   "navbar.artists": [
        |                   ^
      3 |     {
      4 |       "type": 0,
      5 |       "value": "Artists"

      32 |   useAuthIntent()
      33 |   useAuthValidation()
    > 34 |
         | ^
      35 |   const { children, match } = props
      36 |   const routeConfig = match ? findCurrentRoute(match) : null
      37 |   const { isEigen } = useSystemContext()
```